roles/ucp/tasks/main.yml

```diff
@@ -32,6 +32,10 @@
     - "{{ ucp_instance_id_file }}"
   when: node_name != ucp_bootstrap_node_name
 
+- name: copy the ucp license to the remote machine
+  copy: src={{ ucp_license_file }} dest={{ ucp_license_remote }}
```

Comment:
yeah, so user can continue to use it (though I am not sure what they will do with it). 

I think we should add handling for empty `ucp_license_file` name and set it to default. That way we cover the case of a single node unlicensed cluster.